framework/BaseYii.php

```diff
@@ -93,7 +93,7 @@ class BaseYii
      */
     public static function getVersion()
     {
-        return '2.0.49-dev';
+        return '2.0.50-dev';
```

Comment:
Why is it 2.0.50-dev? Isn't this supposed to be 2.2-dev?
Other than this I have no objection

Reply:
done.